plugins/catalog-backend/fixtures/example_components.yaml

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
hey it's maybe a bit nitpicky but can we name this file with a dash instead of an underscore? i know it's currently consistent with the other files, but i would suggest you delete them instead :) as part of this pr actually

Reply:
+1